site/content/blogs/release-v118.en.md

```diff
@@ -22,8 +22,69 @@ enabling you to focus on developing your application, instead of writing deploym
 See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
 ## Certificate import
+
 _Contributed by [Penghao He](https://github.com/iamhopaul123/)_
 
+Developers that have domains managed outside of Route 53 or who want to use HTTPS for Copilot applications without any domain specified can now use a new `--import-cert-arns` flag to import any validated certificates when they create an environment.
+
+```
+$ copilot env init --import-cert-arns arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
+```
+
+For example, the certificate has `example.com` as its domain and `*.example.com` as a subject alternative name (SAN):
+
+```
+$ aws acm describe-certificate --certificate-arn arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012
+
+{
+  "Certificate": {
+    "CertificateArn": "arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012",
+    "DomainName": "example.com",
+    "SubjectAlternativeNames": [
+      "*.example.com"
+    ],
+    "DomainValidationOptions": [
+      {
+        "DomainName": "example.com",
+        "ValidationDomain": "example.com",
+        "ValidationStatus": "SUCCESS",
+        "ResourceRecord": {
+          "Name": "_45c8aa9ac85568e905a6c3852e62ebc6.example.com.",
+          "Type": "CNAME",
+          "Value": "_f8be688050b7d23184863690b3d4baa8.xrchbtpdjs.acm-validations.aws."
+        },
+        "ValidationMethod": "DNS"
+      }
+    ],
```

Comment:
Is this chunk needed for demonstration purpose? Or can it be shortened to reduce noise 🤔 ?

Reply:
I am still more inclined to keep it, because a blog post I think we should have more concrete examples showing them what they should expect for each step.

Reply:
You can also use https://squidfunk.github.io/mkdocs-material/reference/admonitions/?h=colla#collapsible-blocks so that you can toggle to expand or collapse the code block:
```
???+ example "Sample certificate"

    ```json
    
    ```
```